motion.go

```diff
@@ -28,16 +36,16 @@ func (d *MovementDetector) Detect(frame *lepton3.Frame) bool {
 	d1 := absDiffFrames(d.frames[0], d.frames[1])
 	d2 := absDiffFrames(d.frames[1], d.frames[2])
 	m := andFrames(d1, d2)
-	return hasMovement(m)
+	return hasMovement(m, d)
 }
 
-func stripLow(f *lepton3.Frame) *lepton3.Frame {
+func stripLow(f *lepton3.Frame, d *movementDetector) *lepton3.Frame {
```

Comment:
Rather than passing `movementDetector` in `stripLow` and `hasMovement` should become methods of `movementDetector`.